cardcomm/ctkToken/BEIDTokenApp/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -1,63 +1,68 @@
 {
   "images" : [
     {
-      "size" : "16x16",
-      "idiom" : "mac",
       "filename" : "eID 16.png",
-      "scale" : "1x"
+      "idiom" : "mac",
+      "scale" : "1x",
+      "size" : "16x16"
```

Comment:
This file just reorders members. That's unnecessary noise. Please don't add that, or put it in a separate commit if you really must.

Also, please don't commit the icon files together with code. They should be in separate commits.